cmd/server/assets/users/show.html

```diff
@@ -77,6 +81,32 @@ <h1>{{$user.Name}}</h1>
   </main>
 
   {{template "scripts" .}}
+
+  {{if $stats}}
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
push this JS out to a sub-template?

Reply:
The only JS that's "common" is the URL for loader.js, and it didn't seem worth adding that layer of indirection.

Reply:
scrolling back and forth, they looked more similar to me :)